docs/manual/java/guide/production/ConductR.txt

```diff
@@ -0,0 +1,46 @@
+# Lightbend ConductR
+
+ConductR is a container orchestration tool with the main goal of delivering operational productivity. ConductR is also designed to host your Lagom services with resilience.
```

Comment:
I think the word 'container' should be avoided. Probably @huntc et al. can help with the wording in this page.